modelzip/evaluate.py

```diff
@@ -30,7 +30,7 @@ def get_run_cmd(model_dir: Path) -> str:
     Looks for run.py, run.sh, or run (executable)
     """
 
-    run_scripts = [model_dir / "run.py", model_dir / "run.sh", model_dir / "run"]
+    run_scripts = [model_dir / "run.sh", model_dir / "run.py", model_dir / "run"]
```

Comment:
shall we just use `model_dir / "run.sh"` to avoid possible confusion for participants?

Reply:
Surely. I will remove the confusions.